icu4c/source/i18n/unicode/plurrule.h

```diff
@@ -446,6 +454,32 @@ class U_I18N_API PluralRules : public UObject {
                        double *dest, int32_t destCapacity,
                        UErrorCode& status);
 
+#ifndef U_HIDE_DEPRECATED_API
+    /**
+     * Internal-only function that returns FixedDecimals instead of doubles.
+     * Intended for CLDR testing use.
```

Comment:
I would remove or rephrase this line -- CLDR code is all Java, so will never call this C++ method.

Maybe "Only for testing."  ?